.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "wcwave_adaptors"]
-	path = wcwave_adaptors
-	url = https://github.com/tri-state-epscor/wcwave_adaptors.git
```

Comment:
OK. Thanks for the info, I should have provided more info on how to work with wcwave_adaptors as a submodule.

Sounds like you did add changes to wcwave_adaptors? Are they essential to how the vwplatform works?